### ISSUE TITLE
Including styles without webpack

### ISSUE THREAD

**Issue by rikukissa** [zippyui/react-date-picker#32]
Current, it seems that it's not possible to include this module's stylus files from other stylus files without using webpack since you are using webpack's resolve in your @import statements (the ~ char). Also for most people, it's not possible to include the built .css file since it includes normalize.css https://github.com/zippyui/react-date-picker/issues/30

What's the right way of importing the styles without using webpack?

**Comment by rikukissa** [comment]
Also if someone else is struggling with this one, here's a temporary solution

```stylus
@import './node_modules/react-date-picker/node_modules/css-utils/'
@import './node_modules/react-date-picker/style/constants'
@import './node_modules/react-date-picker/style/structure'
@import './node_modules/react-date-picker/style/theme'
```
Presuming that your .styl file is in the same directory with node_modules

**Comment by zippyui-react** [comment]
If you're using `.styl` you can simply import `react-datagrid/style/index.styl` and avoid normalize.css.

Will remove normalize.css in the next version.

**Comment by zippyui-react** [comment]
Yes, that's a solution for now.

Will remove normalize.css in the next version and move the `~css-utils` import from `style/index.styl` to `index.styl` so people will be able to import `react-date-picker/style/index.styl`

**Comment by rikukissa** [comment]
Alright, good to know!

**Comment by zippyui-react** [comment]
We've just released v `2.1.2` to npm, you can now include `react-date-picker/style/index.styl`.
But you'll also want to include `react-date-picker/node_modules/stylus-normalizer/index.styl` (or just include the `stylus-normalizer` package as imported in your own app), which normalizes `display` and `flex` rules in order to support legacy flexbox browsers (safari).

We've also removed dependency on normalize.css. See #30
